internal/duo/duo.go

```diff
@@ -18,20 +18,61 @@ func NewDuoAPI(duoAPI *duoapi.DuoApi) *APIImpl {
 }
 
 // Call call to the DuoAPI.
-func (d *APIImpl) Call(values url.Values, ctx *middlewares.AutheliaCtx) (*Response, error) {
+func (d *APIImpl) Call(values url.Values, ctx *middlewares.AutheliaCtx, method string, path string) (*Response, error) {
```

Comment:
```suggestion
func (d *APIImpl) Call(ctx *middlewares.AutheliaCtx, values url.Values, method string, path string) (*Response, error) {
```

Should we refactor this do ctx is the first param passed?